tika-core/pom.xml

```diff
@@ -55,7 +55,13 @@
       <artifactId>bndlib</artifactId>
       <scope>provided</scope>
     </dependency>
+    <dependency>
```

Comment:
Adding dependency to `tika-core` is serios thing to be discussed in dev@tika.apache.org, IMHO.

Also, please reformat this to use 2 space (no tabs) indents in xml.